install/playbooks/roles/ejabberd/files/renewal-hook.sh

```diff
@@ -3,22 +3,31 @@
 # Exit if jabber server is not running
 systemctl status ejabberd >/dev/null 2>&1 || exit 0
 
-# shellcheck disable=SC1091
 . /etc/homebox/main.cf
 
+do_restart=0
+
 for fqdn in $RENEWED_DOMAINS; do
 
     # Get the subdomain
     sub=$(expr "$fqdn" : '\([^.]*\)')
 
-    if [ "$sub" = "xmpp" ] || [ "$sub" = "conference" ]; then
-        echo "Reloading XMPP ejabberd server"
+    if [ "$sub" = "xmpp" ]; then
         cd "/etc/letsencrypt/live/$DOMAIN"
-
         /bin/cat privkey.pem fullchain.pem > /etc/ejabberd/default.pem
-        chown ejabberd:root /etc/ejabberd/default.pem
-        chmod 640 /etc/ejabberd/default.pem
-        systemctl restart ejabberd
-        break
+        do_restart=1
+    fi
+
+    if [ "$sub" = "conference" ]; then
+        cd "/etc/letsencrypt/live/$DOMAIN"
```

Comment:
Just looking at it again, where does the `$DOMAIN` variable come from? Is it a environment variable from certbot? (must be, because the directories get created alright… Yet, they are created one by one.)

But in this case, if `xmpp` and `conference` are both in the `$RENEWED_DOMAINS` list (does it happen?), what does `$DOMAIN` refer to? We don't want it to be the same in both cases, should it correspond to `$fqdn`?

Sorry, many questions, but I can't find any references to the `$DOMAIN` variable in the certbot docs.
There is a `$RENEWED_LINEAGE` variable for the live directory `/etc/letsencrypt/live/example.com`. But if it happens that both certificates are renewed at the same time and are in the `$RENEWED_DOMAINS` it is no use here, `$fqdn` might be a better option.

What do you think?

Reply:
Ok, it is a list of domains for one certificates. So it concerns the aliases, not separate certificates like here with `xmpp` and `conference`, they won't be listed at the same time.

In that case, the `/etc/letsencrypt/live/$DOMAIN` could apparently be replaced with `$RENEWED_LINEAGE`.

Reply:
For the record, the `$DOMAIN` variable comes from `/etc/homebox/main.cf`.

These comments led to #244 .